code/src/Components/Artist.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+const Artist = (props) => {
+  return (
+    <>
```

Comment:
You don't need this fragment element, because you are returning only one element inside (being precise, this element inside is another fragment, but there it's ok because deeper we have anchor and span :) )